### PR TITLE
added example of passing data to a meld component

### PR DESCRIPTION
**Pull request by olierxleben**
I saw the first issue #20 and wanted to solve it as I just played around with Fask-Meld today. I also just wanted to add value as I really like this project. :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -132,6 +132,45 @@ Counter component.
...
+</div>
+```
+which can be invoked using:
+
+```html
+{# templates/base.html #}
+{% meld 'greeter', name="John Doe" %}
+```
+
+If you want to use the passed arguments from the meld template tag in your component (e.g. configuring the component or addng initial data), you can simply use them from the constructor:
```
Comment (mikeabrahamsen) [inline README.md:158]: Typo "adding"

File: README.md [hunk 2]
```diff
@@ -132,6 +132,45 @@ Counter component.
...
 
+### Pass data to a component
+
+You can, of course, pass data to your meld component. Meld is passing **kwargs 
+to the render function of the *meld* templatetag, so you can pass any number of 
+named arguments. The component is found based on the first parameter, aka name 
+of the component and any number of data passed afterwards. 
+
+Providing a very basic component as an example to display a greeting message using
+the passed value for the keyword "name" or the fallback "Nobody" in its template.
```
Comment (mikeabrahamsen) [inline README.md:143]: The example doubles up on the fallback (it's being set to "Nobody" in the template and in the component, let's remove one of those)

File: README.md (hunk 1 above)
Comment (mikeabrahamsen) [inline README.md:158]: Let's mark this section as "advanced usage" somehow

Comment (olierxleben) [reply README.md]: thx, fixed the typo

Comment (olierxleben) [reply README.md]: added subsection for it

**Review by mikeabrahamsen — APPROVED** [review]

**Comment by mikeabrahamsen** [comment]
This looks great, thank you so much for your help. Cannot even tell you how much I appreciate this!

**Pull request merged**